apistar/routing.py

```diff
@@ -138,11 +138,17 @@ def lookup(self, path: str, method: str) -> RouterLookup:
             raise exceptions.NotFound()
         except werkzeug.exceptions.MethodNotAllowed:
             raise exceptions.MethodNotAllowed()
+        except werkzeug.routing.RequestRedirect:
+            raise exceptions.Found(path + '/')
```

Comment:
Does `RequestRedirect` include something like a url property already?
Can we do, something like this maybe?...

    except werkzeug.routing.RequestRedirect as exc:
        raise exceptions.Found(exc.url)